Added documentation for new flag  in arangoexport (#842)
arangodb/docs: 3.10/release-notes-upgrading-changes310.md

The testcase `hash` was renamed to `persistent-index` to better reflect its
scope.

### arangoexport

> 
> Added new option `--custom-query-bindvars` to arangoexport, so queries given via 
> option `--custom-query` can have bind variables in them.
> Re existing arangoexport option `--query` was renamed to `--custom-query`, for

Reviewer: ```suggestion
The existing arangoexport option `--query` was renamed to `--custom-query`, for
```

The testcase `hash` was renamed to `persistent-index` to better reflect its
scope.

> 
> ### arangobench
> 
> 
> Added new option `--custom-query-bindvars` to arangoexport, so queries given via 
> option `--custom-query` can have bind variables in them. 

Reviewer: ```suggestion

### arangoexport

Re existing arangoexport option `--query` was renamed to `--custom-query`, for
improved naming consistency across different client tools.
Using the old option name `--query` is still supported and will implicitly use the option
`--custom-query` under the hood. Client scripts should eventually be updated to use
the new option name though.
```
